src/test/java/org/haffson/adventofcode/days/day06/CoordinateTest.java

```diff
@@ -0,0 +1,63 @@
+package org.haffson.adventofcode.days.day06;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class CoordinateTest {
+
+    @Test
+    void of_returnSingleCoordinate() {
```

Comment:
Vielleicht darf ich dir bei dieser Gelegenheit `@DisplayName`, empfehlen, das wir in NLP überall nutzen.
Es führt dazu, dass in der Test-Übersicht direkt lesbare Inhalte sehen.

Beispiel:
![image](https://user-images.githubusercontent.com/32487587/182154478-72a06a10-d985-422b-874d-f89c56685390.png)

![image](https://user-images.githubusercontent.com/32487587/182154534-b1f61853-ea1c-42fb-aa13-beeb35db2c80.png)




```suggestion
    @DisplayName("of() should return single coordinate")
    void of_returnSingleCoordinate() {
```

Reply:
Stimmt, das hattest du mir gezeigt, als ich dir zugeschaut habe - wird jetzt genutzt!